### PR TITLE
Profile Page gui + auth service test 

### PR DESCRIPTION
**Pull request by JonasEbbinghaus25**
profile data on the profile page cannot yet be changed

### CODE REVIEW

File: src/main/web/src/organisms/login/Login.tsx [hunk 1]
```diff
@@ -121,15 +121,17 @@ const Login: React.FC<Props> = () => {
                             </div>
 
                             <div className="google-oauth-login">
```
Comment (maxschwinghammer) [inline src/main/web/src/organisms/login/Login.tsx:123]: Google OAuth Provider is not necessary here, it is already used in App.tsx

File: src/main/web/src/scenes/Profile/ProfilePage.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+interface ProfileData {
+    username: string;
+    email: string;
+    followers: number;
+    profilePicture: string;
+}
+
+export const ProfilePage: React.FC = () => {
+    const [profileData, setProfileData] = useState<ProfileData>({
+        username: "Dieter B.",
```
Comment (maxschwinghammer) [inline src/main/web/src/scenes/Profile/ProfilePage.tsx:16]: Use actual data for the profile data

File: src/main/web/src/scenes/Profile/ProfilePage.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
...VkVDQWxCN2UyZ1FJRUNCQUlDa2dRSkw2WmhNZ1FJQUFnVklCQVZKNmVHc1RJRUNBQUlHa2dBQko2cHROZ0FBQkFnUktCUVJJNmVHdFRZQUFBUUlFa2dJQ0pLbHZOZ0VDQkFnUUtCVVFJS1dIdHpZQkFnUUlFRWdLQ0pDa3Z0a0VDQkFnUUtCVVFJQ1VIdDdhQkFnUUlFQWdLU0JBa3ZwbUV5QkFnQUNCVWdFQlVucDRheE1nUUlBQWdhU0FBRW5xbTAyQUFBRUNCRW9GQkVqcDRhMU5nQUFCQWdTU0FnSWtxVzgyQVFJRUNCQW9GUkFncFllM05nRUNCQWdRU0FvSWtLUysyUVFJRUNCQW9GUkFnSlFlM3RvRUNCQWdRQ0FwSUVDUyttWVRJRUNBQUlGU0FRRlNlbmhyRXlCQWdBQ0JwSUFBU2VxYlRZQUFBUUlFU2dVRVNPbmhyVTJBQUFFQ0JKSUNBaVNwYnpZQkFnUUlFQ2dWRUNDbGg3YzJBUUlFQ0JCSUNnaVFwTDdaQkFnUUlFQ2dWRUNBbEI3ZTJnUUlFQ0JBSUNrZ1FKTDZaaE1nUUlBQWdWSUJBVko2ZUdzVElFQ0FBSUdrZ0FCSjZwdE5nQUFCQWdSS0JRUkk2ZUd0VFlBQUFRSUVrZ0lDSktsdk5nRUNCQWdRS0JVUUlLV0h0ellCQWdRSUVFZ0tDSkNrdnRrRUNCQWdRS0JVUUlDVUh0N2FCQWdRSUVBZ0tTQkFrdnBtRXlCQWdBQ0JVZ0VCVW5wNGF4TWdRSUFBZ2FTQUFFbnFtMDJBQUFFQ0JFb0ZCRWpwNGExTmdBQUJBZ1NTQWdJa3FXODJBUUlFQ0JBb0ZSQWdwWWUzTmdFQ0JBZ1FTQW9Ja0tTKzJRUUlFQ0JBb0ZSQWdKUWUzdG9FQ0JBZ1FDQXA4QWlRNHppKzFscTN2ZmNrZjVIWkJBZ1FJRUNBd1BzTHpNeGVhOTMzZVo2Zk0zTzdWdDc3K3M2SEFBRUNCQWdRSVBCOGdabmZkeDE3NzU4QXVmL1Z5UE1uZVNJQkFnUUlFQ0JBNEovQTlhL0xGU0FmVjR6UUlVQ0FBQUVDQkFpOFNHQytBVmxHWkcxdDF6OVBBQUFBQUVsRlRrU3VRbUNDIi8+CjwvZGVmcz4KPC9zdmc+Cg==" // Verwende den tatsächlichen Pfad
```
Comment (maxschwinghammer) [inline src/main/web/src/scenes/Profile/ProfilePage.tsx:19]: Use useState for the profile picture and fetch the image from ProfilePictureService

File: src/main/web/src/scenes/Profile/ProfilePage.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
+.profile-page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vh;
+  font-family: "Helvetica Neue", Arial, Helvetica, sans-serif;
+}
+
+.profile-container {
+  background-color: #2b2b2b;
```
Comment (maxschwinghammer) [inline src/main/web/src/scenes/Profile/ProfilePage.css:10]: User var(--component-grey) for background

File: src/main/web/src/scenes/Profile/ProfilePage.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
+.profile-page {
```
Comment (maxschwinghammer) [inline src/main/web/src/scenes/Profile/ProfilePage.css:1]: Div `profile-page` is not necessary, `page` already determines everything from this, the height is not required

File: src/main/web/src/scenes/Profile/ProfilePage.tsx [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    };
+
+    const handlePasswordChange = () => {
+
+    };
+
+    return (
+        <div className="page">
+            <Header/>
+            <div className="profile-page">
```
Comment (maxschwinghammer) [inline src/main/web/src/scenes/Profile/ProfilePage.tsx:44]: Div `profile-page` is not necessary, `page` already determines everything from this

File: src/main/web/src/scenes/Profile/ProfilePage.tsx [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+                <div className="profile-container">
+                    <div className="profile-picture">
+                        <img src={profileData.profilePicture} alt="Profile"/>
```
Comment (maxschwinghammer) [inline src/main/web/src/scenes/Profile/ProfilePage.tsx:47]: Image is not centered

**Review by maxschwinghammer — CHANGES_REQUESTED** [review]
See comments above

File: src/main/web/src/scenes/Profile/ProfilePage.tsx (hunk 2 above)
Comment (JonasEbbinghaus25) [reply src/main/web/src/scenes/Profile/ProfilePage.tsx:16]: not part of this pr

File: src/main/web/src/organisms/login/Login.tsx (hunk 1 above)
Comment (JonasEbbinghaus25) [reply src/main/web/src/organisms/login/Login.tsx:123]: I have not added but only taken over

File: src/main/web/src/scenes/Profile/ProfilePage.tsx (hunk 3 above)
Comment (JonasEbbinghaus25) [reply src/main/web/src/scenes/Profile/ProfilePage.tsx:19]: the pr is only there to provide the user interface. The functionality comes with a later pr

**Comment by denniskp** [comment]
Bitte Squash Merge das am Ende dann, lol

**Review by maxschwinghammer — CHANGES_REQUESTED** [review]
Please fix React build

**Pull request closed without merging**